src/http/user-image-http.js

```diff
@@ -0,0 +1,69 @@
+// # imagesService
+//      everything related to image manipulation
+//
+'use strict';
+
+import * as gcs from '../util/gcs';
+import * as userImageCore from '../core/user-image-core';
+import {decodeBase64Image} from '../util/base64';
+import { processImage } from '../util/image-processor';
+
+const logger = require('../util/logger')(__filename);
+const uuidV1 = require('uuid/v1');
+
+const ALLOWED_MIME_TYPES = new Set(['image/jpeg', 'image/gif', 'image/png']);
+
+function uploadImage(imageName, imageFile, imageOpts) {
+  logger.info('Uploading', imageName);
+
+  return Promise.resolve()
```

Comment:
No need to do an "empty" promise here, just do synchronic calls first and return the start of the Promise chain